xla/service/gpu/cudnn_fused_conv_rewriter_test.cc

```diff
@@ -295,6 +319,105 @@ TEST_F(CudnnFusedConvRewriterTest, TestBias) {
     })");
 }
 
+TEST_F(CudnnFusedConvRewriterTest, TestBiasMultiCall) {
+
+  // max(0, conv(x, w) + bias);
+  std::string code = R"(
+    HloModule Test
+
+    ENTRY Test {
+      zero = TYPE[] constant(0)
+      zeros = TYPE[1,<<<format>>>,64] broadcast(zero), dimensions={}
+
+      input = TYPE[1,<<<format>>>,64] parameter(0)
+      filter = TYPE[3,3,64,64] parameter(1)
+      bias = TYPE[64] parameter(2)
+
+      conv = TYPE[1,<<<format>>>,64] convolution(input, filter), window={size=3x3 pad=1_1x1_1}, dim_labels=b01f_01io->b01f, feature_group_count=1
+      broadcasted_bias = TYPE[1,<<<format>>>,64] broadcast(bias), dimensions={3}
+      add1 = TYPE[1,<<<format>>>,64] add(conv, broadcasted_bias)
+      ROOT relu = TYPE[1,<<<format>>>,64] maximum(zeros, add1)
+    })";
+    absl::flat_hash_map<absl::string_view, absl::string_view> replacements;
+    replacements["<<<format>>>"] = "3,3";
+    TestMatchWithAllTypes(absl::StrReplaceAll(code, replacements));
+    replacements["<<<format>>>"] = "5,5";
+    TestMatchWithAllTypes(absl::StrReplaceAll(code, replacements));
+    replacements["<<<format>>>"] = "3,3";
+    TestMatchWithAllTypes(absl::StrReplaceAll(code, replacements));
+}
+
+TEST_F(CudnnFusedConvRewriterTest, TestBiasMultiCallMultiThread) {
+  // max(0, conv(x, w) + bias);
+  std::string code = R"(
+    HloModule Test
+
+    ENTRY Test {
+      zero = TYPE[] constant(0)
+      zeros = TYPE[1,<<<format>>>,64] broadcast(zero), dimensions={}
+
+      input = TYPE[1,<<<format>>>,64] parameter(0)
+      filter = TYPE[3,3,64,64] parameter(1)
+      bias = TYPE[64] parameter(2)
+
+      conv = TYPE[1,<<<format>>>,64] convolution(input, filter), window={size=3x3 pad=1_1x1_1}, dim_labels=b01f_01io->b01f, feature_group_count=1
+      broadcasted_bias = TYPE[1,<<<format>>>,64] broadcast(bias), dimensions={3}
+      add1 = TYPE[1,<<<format>>>,64] add(conv, broadcasted_bias)
+      ROOT relu = TYPE[1,<<<format>>>,64] maximum(zeros, add1)
+    })";
+    absl::flat_hash_map<absl::string_view, absl::string_view> replacements;
+    replacements["<<<format>>>"] = "3,3";
+    replacements["TYPE"] = "f16";
+    const std::string hlo = absl::StrReplaceAll(code, replacements);
```

Comment:
Since you are using the replacements just one, consider to just hard-code the test fixture.